### PR TITLE
avm-staging getting started with python updated

### PR DESCRIPTION
**Pull request by iskysun96**
added the getting started with python page.

### CODE REVIEW

File: docs/sdks/python/index.md [hunk 1]
```diff
@@ -1 +1,313 @@
...
+
+If you are a visual learner, try our submitting your first transaction [live demo](https://replit.com/@Algorand/gettingStartedPython#main.py) or watch a [full video](https://www.youtube.com/watch?v=ku2hFalMWmA) that explains the following steps.
+
+# Sandbox Install
+
+!!! Prerequisites
+    - Docker Compose ([install guide](https://docs.docker.com/compose/install/))
+    - Git ([install guide](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)) 
+
+Algorand provides a docker instance for setting up a node, which can be used to get started developing. To install and use this instance, follow these instructions.
```
Comment (ejbaran) [inline docs/sdks/python/index.md:15]: maybe: "which can be used to get started developing quickly."

File: docs/sdks/python/index.md [hunk 2]
```diff
@@ -1 +1,313 @@
...
\ No newline at end of file
+title: Your First Transaction
+
+This section is a quick start guide for interacting with the Algorand network using Python. This guide will help to install ***sandbox***, which provides a node for testing and development. This guide will also help to install the Python SDK, create an account and submit your first transaction on Algorand. 
+
+# Alternative Guide
+
+If you are a visual learner, try our submitting your first transaction [live demo](https://replit.com/@Algorand/gettingStartedPython#main.py) or watch a [full video](https://www.youtube.com/watch?v=ku2hFalMWmA) that explains the following steps.
+
+# Sandbox Install
```
Comment (ejbaran) [inline docs/sdks/python/index.md:9]: Call this "Install Sandbox" to mirror the other titles which are "do x" style.

File: docs/sdks/python/index.md [hunk 3]
```diff
@@ -1 +1,315 @@
...
+    private_key, address = account.generate_account()
+    print("My address: {}".format(address))
+    print("My private key: {}".format(private_key))
+    print("My passphrase: {}".format(mnemonic.from_private_key(private_key)))
+```
+[`Watch Video`](https://youtu.be/ku2hFalMWmA?t=161)  
+[`More Information`](https://developer.algorand.org/docs/features/accounts/create/#standalone)
+
+!!! Tip
+    Make sure to save your account's address and passphrase at a seperate place, as they will be used later on.
```
Comment (ejbaran) [inline docs/sdks/python/index.md:77]: "at a seperate" --> "in a separate"

File: docs/sdks/python/index.md [hunk 4]
```diff
@@ -1 +1,315 @@
...
+​
+!!! Warning
+    Never share Mnemonic and private key. Production environments require stringent private key management. For more information on key management in community Wallets, click [here](https://developer.algorand.org/docs/community/#wallets). For the [Algorand open source wallet](https://developer.algorand.org/articles/algorand-wallet-now-open-source/), click [here](https://github.com/algorand/algorand-wallet).
```
Comment (ejbaran) [inline docs/sdks/python/index.md:80]: lower case "mnemonic"

File: docs/sdks/python/index.md [hunk 5]
```diff
@@ -1 +1,315 @@
...
+    algod_token = "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa"
+    algod_client = algod.AlgodClient(algod_token, algod_address)
+```
+[`Watch Video`](https://youtu.be/ku2hFalMWmA?t=322)  
+
+!!! Info
+    The example code connects to the sandbox Algod client. If you want to connect to a Purestake client, see Purestake's [code samples](https://developer.purestake.io/code-samples).
+
+# Check Your Balance
+Before moving on to the next step, make sure your account has been funded by the faucet.
```
Comment (ejbaran) [inline docs/sdks/python/index.md:107]: Maybe: Before moving on to the next step, visit the TestNet faucet to fund your account.

File: docs/sdks/python/index.md [hunk 6]
```diff
@@ -1 +1,315 @@
...
+
+```python
+    account_info = algod_client.account_info(my_address)
+    print("Account balance: {} microAlgos".format(account_info.get('amount')) + "\n")
+```
+[`Watch Video`](https://youtu.be/ku2hFalMWmA?t=381) 
+
+# Build First Transaction
+Communication with the Algorand network is performed using transactions. To create a payment transaction use the following code.
```
Comment (ejbaran) [inline docs/sdks/python/index.md:116]: Maybe: Use transactions to communicate with the Algorand network.

**Review by ejbaran — CHANGES_REQUESTED** [review]
- Use relative links for directs to pages on the portal.
- For all the see-more-detail type links, we should have that open as new tabs. Use the following syntax: `[text](link){target=_blank}`
- We reference the indexer but I don't think we should unless we are doing the whole thing in a private network. Think this should either all be on a private net or all on testnet.

File: docs/sdks/python/index.md (hunk 6 above)
Comment (barnjamin) [reply docs/sdks/python/index.md:116]: consider replacing communicate with interact to emphasize an effect?

File: docs/sdks/python/index.md [hunk 7]
```diff
@@ -1 +1,315 @@
...
+```
+[`Run Code`](https://replit.com/@Algorand/gettingStartedPython#main.py)  
+[`Watch Video`](https://youtu.be/ku2hFalMWmA?t=556)  
+[`Go to Github`](https://github.com/algorand/docs/blob/staging/examples/start_building/v2/python/your_first_transaction.py)
+​​
+!!! Warning 
+    In order for this transaction to be successful, the account must be funded. 
+
+# Viewing the Transaction
+To view the transaction, open the [AlgoExplorer](https://testnet.algoexplorer.io/) or [Goal Seeker](https://goalseeker.purestake.io/algorand/testnet) and paste the transaction ID into the search bar.
```
Comment (ejbaran) [inline docs/sdks/python/index.md:304]: "open the" --> "open"

File: docs/sdks/python/index.md [hunk 8]
```diff
@@ -1 +1,315 @@
...
+first_transaction_example(private_key, my_address)​
+```
+[`Run Code`](https://replit.com/@Algorand/gettingStartedPython#main.py)  
+[`Watch Video`](https://youtu.be/ku2hFalMWmA?t=556)  
+[`Go to Github`](https://github.com/algorand/docs/blob/staging/examples/start_building/v2/python/your_first_transaction.py)
+​​
+!!! Warning 
+    In order for this transaction to be successful, the account must be funded. 
+
+# Viewing the Transaction
```
Comment (ejbaran) [inline docs/sdks/python/index.md:303]: "View the Transaction"  to be consistent with previous titles.

File: docs/sdks/python/index.md [hunk 9]
```diff
@@ -1 +1,315 @@
...
+To view the transaction, open the [AlgoExplorer](https://testnet.algoexplorer.io/) or [Goal Seeker](https://goalseeker.purestake.io/algorand/testnet) and paste the transaction ID into the search bar.  
+
+[`Watch Video`](https://youtu.be/ku2hFalMWmA?t=618) 
+
+# Setting Up Your Editor/Framework
```
Comment (ejbaran) [inline docs/sdks/python/index.md:308]: "Set up" for consistency

File: docs/sdks/python/index.md (hunk 5 above)
Comment (iskysun96) [reply docs/sdks/python/index.md:107]: we do this in the "fund the account" section

**Comment by iskysun96** [comment]
Need to update wait for confirmation link (line 140)
- don't have a page going over wait for confirmation anymore.

Need to update community wallet link (line 71)
- couldn't find the community page in avm-staging

**Review by ejbaran — APPROVED** [review]

**Pull request merged**